plugins/PostProcessingPlugin/scripts/DisplayRemainingTimeOnLCD.py

```diff
@@ -0,0 +1,86 @@
+
+from ..Script import Script
+
+import re
+import datetime
+
+
+class DisplayRemainingTimeOnLCD(Script):
+
+    def __init__(self):
+        super().__init__()
+
+
+    def getSettingDataString(self):
+        return """{
+            "name":"Disaplay Remaining Time on LCD",
+            "key":"DisplayRemainingTimeOnLCD",
+            "metadata": {},
+            "version": 2,
+            "settings":
+            {
+                "TurnOn":
+                {
+                    "label": "Enable",
+                    "description": "When enabled, It will write Time Left HHMMSS on the display",
+                    "type": "bool",
+                    "default_value": true
+                }
+            }
+        }"""
+
+    def execute(self, data):
+        if self.getSettingValueByKey("TurnOn"):
+            TotalTime = 0           # Var for total time
```

Comment:
According to our codestyle, this should be total_time (as it's a variable, it should use _ in between words and be in lower cases). 
It's good that you put comments in your code, but this one is just repeating what's already there (when in doubt, try to write comments that explain why you do something. You should only add comments about the "what" if it is really difficult (or it's a trick or something).

Reply:
Thanks for pointing that out, is was not aware of that. The names should be fixed now.